scripts/build-json/build-sidebars.js

```diff
@@ -0,0 +1,104 @@
+const fs = require('fs');
+const path = require('path');
+const yaml = require('js-yaml');
+const matter = require('gray-matter');
+
+const sidebarDir = path.join(process.cwd(), 'content', 'sidebars');
+const packagedDir = path.join(process.cwd(),'packaged', 'sidebars');
+
+/**
+ * Get a single sidebar item from front matter
+ */
+function sidebarItemFromFile(filePath) {
+  const {data} = matter(fs.readFileSync(filePath, 'utf8'));
+  return {
+    title: data.title,
+    mdn_url: data.mdn_url
+  };
+}
+
+/**
+ * Get a single sidebar item from meta.yaml
+ */
+function sidebarItemFromMeta(metaPath) {
+  const meta = yaml.safeLoad(fs.readFileSync(metaPath, 'utf8'));
+  return {
+    title: meta.title,
+    mdn_url: meta['mdn-url']
```

Comment:
This is gross obviously. Should be mdn_url (or slug as has been discussed elsewhere).